docs/BEMAN_STANDARD.md

```diff
@@ -58,18 +58,23 @@ recommendations.
 
 ## License
 
-**[LICENSE.APPROVED]** REQUIREMENT: All Beman libraries must be licensed
-under an approved license. These are:
+### **[LICENSE.APPROVED]** REQUIREMENT
+
+All Beman libraries must be licensed under an approved license. These are:
```

Comment:
```suggestion
REQUIREMENT: All Beman libraries must be licensed under an approved license. These are:
```

I think this format is more concise. I would keep just `**[LICENSE.APPROVED]**` in the tittle